src/org/daisy/dotify/formatter/impl/obfl/ObflParserImpl.java

```diff
@@ -1641,8 +1641,6 @@ private void parseVolumeTemplate(
             OBFLVariable.VOLUME_COUNT,
             OBFLVariable.SHEET_COUNT,
             OBFLVariable.VOLUME_SHEET_COUNT);
-        condition.setVariableName(OBFLVariable.VOLUME_COUNT, getAttr(event, "volume-count-variable"));
-        condition.setVariableName(OBFLVariable.VOLUME_NUMBER, getAttr(event, "volume-number-variable"));
```

Comment:
@PaulRambags Isn't this a regression?

Reply:
Hmm I see you created the issue https://github.com/mtmse/obfl/issues/13. Fine by me. But maybe a warning should be created when the attribute is specified. By the way there is another comment (a FIXME) somewhere in this file about it.

Reply:
Good point. I'll throw an `UnsupportedOperationException`. This exception could be removed from the code in a later stadium.

Reply:
Great. Can you also add the exception [here](https://github.com/mtmse/dotify.formatter.impl/blob/ba9320a09f95ecc67741f1a4e116c576c585320f/src/org/daisy/dotify/formatter/impl/obfl/ObflParserImpl.java#L267-L268)?

https://github.com/mtmse/dotify.formatter.impl/blob/ba9320a09f95ecc67741f1a4e116c576c585320f/src/org/daisy/dotify/formatter/impl/obfl/ObflParserImpl.java#L267-L268

Actually I have been using this attribute in Pipeline, being totally unaware it is ignored. Luckily I just set it to the default value "page" :).

Reply:
>Actually I have been using this attribute in Pipeline, being totally unaware it is ignored.
Beware! In the coming release, an exception will be thrown even if you use the default value. ;-)

Reply:
Yes, sure, I'm removing it.